src/components/modaldialog.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div uk-modal>
+    <div class="uk-modal-dialog uk-modal-body">
+      <keep-alive><component
```

Comment:
Do you need `<keep-alive>` here?

Reply:
Probably not. It is a caching mechanism, but probably doesn't need to be there